src/Pages/Admin/Admin.js

```diff
@@ -10,39 +9,8 @@ export class Admin extends Component {
     }
   }
 
-  componentDidMount () {
-    const token = window.localStorage
-      ? window.localStorage.getItem('jwtToken')
-      : ''
-
-    // Immediately direct to /login if no jwtToken token present
-    if (!token) {
-      if (this.props.history) this.props.history.push('/login')
-      return
-    }
-
-    // Verify if token is valid
-    // As user persmissions are created, the verify auth should be more extensive
-    // and return views as the permissions defines
-    axios
-      .post('/api/user/verify', { token })
-      .then(res => {
-        this.setState({
-          user: res.data
-        })
-
-        // If the user doesn't have sufficient privilages, send them to their profile page
-        if (res.data.accessLevel < 2 && this.props.history) {
-          this.props.history.push('/dashboard')
-        }
-      })
-      .catch(() => {
-        if (this.props.history) this.props.history.push('/login')
-      })
-  }
-
   render () {
-    return <div>{this.state.user.accessLevel >= 2 && <div>Admin</div>}</div>
```

Comment:
this is necessary to enable access to profile and admin in stead of the login button. See the last section of my comment.

Reply:
I removed this as the `<PrivateRoute />` component takes care of this for you. In Routing.js, we see the Admin component get rendered on [this condition](https://github.com/SCE-Development/Core-v4/blob/aef4801af88c32c34bad2aba7ba09319db3e183a/src/Routing.js#L31). A [test](https://github.com/SCE-Development/Core-v4/blob/aef4801af88c32c34bad2aba7ba09319db3e183a/test/frontend/Routing.test.js#L217) also proves that with insufficient priviledges the user cannot see admin.